CMakeLists.txt

```diff
@@ -180,7 +180,7 @@ ExternalProject_Add(
 
   PREFIX          BufferTree
   GIT_REPOSITORY  "https://github.com/Victor-C-Zhang/FastBufferTree.git"
-  GIT_TAG         "main"
+  GIT_TAG         "circular_queue"
```

Comment:
remember to change this once we merge to main

Reply:
Yep!